Emberjs/ember-auth/test-app/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "test-app",
+  "version": "4.1.0",
+  "private": true,
+  "scripts": {
+    "build": "ember build",
+    "lint": "eslint app config server tests *.js",
+    "start": "ember serve",
+    "start:fastboot": "NODE_ENV=production node fastboot-server.js",
+    "start:demo-api": "node server/demo-api.js",
+    "test": "ember test",
+    "test:all": "ember try:each",
+    "test:one": "ember try:one",
+    "test:fastboot": "ember fastboot:test"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "^1.3.0",
+    "@glimmer/component": "^1.0.0",
+    "@glimmer/tracking": "^1.0.0",
+    "babel-eslint": "^10.1.0",
+    "body-parser": "^1.19.0",
+    "bootstrap": "^4.6.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "chai": "^4.1.0",
+    "cors": "^2.8.5",
+    "ember-cli": "~3.17.0",
+    "ember-cli-app-version": "^3.2.0",
+    "ember-cli-babel": "^7.19.0",
+    "ember-cli-chai": "^0.5.0",
+    "ember-cli-content-security-policy": "~1.1.1",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-fastboot": "^2.2.1",
+    "ember-cli-fastboot-testing": "^0.4.0",
+    "ember-cli-htmlbars": "^4.2.3",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-mocha": "^0.15.0-beta.1",
+    "ember-cli-pretender": "^3.0.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^3.0.0",
+    "ember-data": "~3.17.0",
+    "ember-disable-prototype-extensions": "^1.1.2",
+    "ember-engines": "^0.8.5",
+    "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^8.0.1",
+    "ember-load-initializers": "^2.1.1",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-mocha": "^0.16.2",
+    "ember-resolver": "^7.0.0",
+    "ember-simple-auth": "4.1.0",
+    "ember-source": "~3.17.0",
+    "ember-source-channel-url": "^1.1.0",
+    "ember-try": "^1.1.0",
+    "eslint": "^5.16.0",
+    "eslint-config-simplabs": "^0.4.0",
+    "eslint-plugin-ember": "^8.1.1",
+    "eslint-plugin-mocha": "^6.1.1",
+    "eslint-plugin-node": "^11.0.0",
+    "express": "^4.17.1",
+    "fastboot-app-server": "^2.0.0",
+    "glob": "^7.1.6",
+    "handlebars": "~4.7.1",
+    "loader.js": "^4.7.0",
+    "mocha": "^7.0.1",
+    "my-engine": "./packages/test-app/lib/my-engine",
```

Comment:
the path for `"my-engine"` here is incorrect and will cause an error when doing `yarn install` - correct path is `"./lib/my-engine"`